spec/lib/api/1.1/skus-spec.js

```diff
@@ -16,7 +17,9 @@ describe('Http.Api.Skus', function () {
         ]);
         return helper.startServer([
             dihelper.simpleWrapper(workflowApiService, 'Http.Services.Api.Workflows')
-        ]);
+        ]).then(function() {
+            _ = helper.injector.get('_');
```

Comment:
The design you want reinforced is to stash global variables in another repo with no documentation or comments that you are doing so, and then expect people to use them?  

Should I use that pattern more?  Should I be stashing globals in on-core for other things and not using the injector as an interface?  I use 'fs' sometimes.  Should I put it in a global in on-core?